fighthealthinsurance/common_view_logic.py

```
@@ -1399,6 +1429,8 @@
                     "Failed to save proposed appeal: {e}"
                 )
                 pass
+            passed = time.time() - t
```

Ruff (0.8.2)

`time` may be undefined, or defined from star imports

(F405)